src/js/Rickshaw.Graph.RangeSlider.Preview.js

```diff
@@ -137,10 +148,7 @@ Rickshaw.Graph.RangeSlider.Preview = Rickshaw.Class.create({
       var graph = new Rickshaw.Graph(graphArgs)
       self.previews.push(graph)
 
-      parent.onUpdate(function() {
-        graph.render()
-        self.render()
-      })
+      parent.onUpdate(self.render())
```

Comment:
No bueno - this is going to immediately invoke `render`, as you're now passing the *invoked* render function to `onUpdate` - this should be: `parent.onUpdate(self.render)`

Reply:
Fixed for real now.